datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -1433,3 +1443,22 @@ func resourceDatadogSecurityMonitoringRuleDelete(ctx context.Context, d *schema.
 
 	return nil
 }
+
+func resourceDatadogSecurityMonitoringRuleCustomizeDiff(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	if validate, ok := diff.GetOkExists("validate"); ok && !validate.(bool) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏If the value does not exist, I think you should skip the validation (to be retrocompatible with existing behaviour).

Reply:
My bad, there were some glitches regarding the diff. For some tests, it wasn't reaching CustomizeDiff stage or in same case the value wasn't properly coming in ResourceDiff.